openquake/commonlib/logictree.py

```diff
@@ -558,20 +591,45 @@ def __init__(self, filename, validate=True, seed=0, num_samples=0,
         except AttributeError:
             raise ValidationError(
                 root, self.filename, "missing logicTree node")
-        self.apply_to_sources = set()
         self.parse_tree(tree, validate)
 
     def on_each_source(self):
         """
         :returns: True if the logic tree is defined on each source
```

Comment:
I think this comment must be updated since now you return a set

Reply:
Actually I am still returning a boolean here, `self.info.applytosources and self.info.applytosources == self.source_ids`